assets/css/index.css

```diff
@@ -227,6 +227,7 @@ hr {
   word-wrap: break-word;
   font-family: 'Syne', sans-serif;
 } 
+
```

Comment:
This file shouldn't be changed. Return it to how it was with `git checkout assets/css/index.css`